packages/design-system/src/components/SearchBar/style.scss

```
@@ -0,0 +1,62 @@
+@import '../../scss/variables/colors';
+
+$base-class: "search-bar";
+
+.#{$base-class}__container {
+  display: inline-flex;
+  position: relative;
+  flex-direction: column;
+  justify-content: center;
+  height: 32px;
+  width: 100%;
+}
+
+.#{$base-class}__icon {
+  position: absolute;
+  align-self: center;
+
+  &--search {
+    @extend .#{$base-class}__icon;
+    left: 0;
+    padding: 7px 13px;
+    fill: $icon-primary-color;
+
+    &-compact {
+      cursor: pointer;
+    }
+  }
+
+  &--clear {
+    @extend .#{$base-class}__icon;
+    right: 13px;
+    cursor: pointer;
+    fill: $icon-primary-color;
+  }
```

```suggestion
  @include icon-button-interactive-styles;
  }
```

this will reset browser specific styles and gives us control how the button looks